dotnet/Devolutions.NowClient.Cli/Program.cs

```diff
@@ -244,4 +250,71 @@ private static async Task ExecutePowerShellCommand(NowClient client, string comm
             throw;
         }
     }
+
+    /// <summary>
+    /// Executes the RDM run command - starts an RDM application.
+    /// </summary>
+    private static async Task ExecuteRdmRunCommand(NowClient client)
```

Comment:
Added some test commands blindly to speed up adding those on Devolutions agent side later down the line.

Reply:
awesome, great idea!